### DISCUSSION TITLE
`striped` option

### DISCUSSION THREAD

**Discussion by mcamou** [Sithuk/ubuntu-server-zfsbootmenu#27] [General]
I have 2 SSDs in my laptop that I stripe across (not enough storage to mirror). It would be good to have an option for striping across devices, apart from mirroring and raidz.

**Comment by Sithuk** [comment]
As you say, you can setup your two drives in a mirror or raidz1 configuration. What layout do you have in mind?

**Reply by mcamou** [reply]
With raidz1 and mirror, if you have 2x1TB SSD's you get 1TB usable space. I want 2TB of usable space in the volume, no redundancy. This is the default for `zpool create`, as in `zpool create tank /dev/sda1 /dev/sdb1`. Not too useful for a server since a single-disk failure will destroy the whole pool, but good for a workstation or laptop as long as you have good backups.

**Reply by Sithuk** [reply]
I've just updated the script for raid0 support. I'd welcome any testing you could do or feedback you could provide.
